DQM/DTMonitorModule/python/dtDQMOfflineSources_Cosmics_cff.py

```diff
@@ -42,11 +46,12 @@
 from DQM.DTMonitorModule.dtTriggerEfficiencyTask_cfi import *
 
 dtSourcesCosmics = cms.Sequence(dtDataIntegrityUnpacker  +
-                                DTDataIntegrityTask +
-                                dtDCSByLumiMonitor + 
-                                dtRunConditionVar + 
-                                dtSegmentAnalysisMonitor +
-                                dtResolutionAnalysisMonitor +
-                                dtEfficiencyMonitor +
-                                dtTriggerEfficiencyMonitor +
-                                dqmInfoDT)
+                         twinMuxStage2Digis +
```

Comment:
Hi @davidlange6 , @rekovic but, I'm not able to find where should I move the twiMuxStage2Digis in this DQM sequence. Could you please point me to the right package? 

I can trace back to see that DQMOffline/Configuration/python/DQMOffline_cff.py is calling this config file, but I ignore where all the raw2digi is being set for DQM, if so. Thanks